code/modules/antagonists/wrestler/wrestler.dm

```diff
@@ -9,9 +9,43 @@
 
 	give_equipment(fake_equipment = FALSE)
 		src.fake = fake_equipment
+		src.owner.current.add_wrestle_powers(fake_equipment)
 		if (ismobcritter(src))
 			display_name = "wrestledoodle"
-		src.owner.current.add_wrestle_powers(fake_equipment)
+			return
+
+		// Assign wrestle attire.
+		if (ishuman(src.owner.current))
```

Comment:
Oh that's fine too I think, was initially meaning you could also do if (!ishuman(src.owner.current)) return